spec/core/finalization/schemas/FinalizationProofDTO.yml

```diff
@@ -0,0 +1,23 @@
+type: object
+required:
+  - version
+  - finalizationEpoch
+  - finalizationPoint
+  - height
+  - hash
+  - messageGroups
+properties:
+  version:
+    type: integer
+  finalizationEpoch:
+    type: integer
```

Comment:
Why is `finalizationEpoch` an integer here, whereas in [/schemas/FinalizationEpoch.yml](https://github.com/nemtech/symbol-openapi/blob/main/spec/schemas/FinalizationEpoch.yml) it is an int64?

Wouldn't it be nicer to reference FinalizationEpoch.yml here?

Reply:
this is why i poked fernando, i'm unsure whether he needs to use int64 for integers or just integer works out (due to a java generator thing)

Reply:
The ref to the tyle files will do the trick for both epoch and point. height too?

Reply:
I would much prefer the ref since then we have descriptions and examples all in the same place.

Reply:
@fboucquez epoch and point are uint32 (height is uint64), not sure which openapi type or ref you need

Reply:
Ref is already in height to the Height.yml file, missed that. All good.  Just the other 2

Reply:
should we use the `Uint32` schema for point and epoch? just saw there's one

Reply:
i just saw there's an `Uint32` schema, and that this schema is defined with int64

in the same manner, point and epoch have its own schemas defined with int64, isn't it the same and therefore everything is fine?

Reply:
@fboucquez